src/client/sandbox/shadow-ui.js

```diff
@@ -64,6 +64,32 @@ export default class ShadowUI extends SandboxBase {
         return ShadowUI._filterList(styleSheetList, item => ShadowUI._filterElement(item.ownerNode));
     }
 
+    static _filterFirstNonShadowElement (nodeList) {
+        for (var i = 0; i < nodeList.length; i++) {
+            var el = ShadowUI._filterElement(nodeList[i]);
+
+            if (el)
+                return nodeList[i];
+        }
+
+        return null;
+    }
+
+    static _createWrapperForQuerySelectorFn (nativeQuerySelectorFnName, nativeQuerySelectorAllFnName) {
```

Comment:
We can't pass the native functions instead of string names because it saves in closure and we can't test them with the `checkNativeFunctionArgs` function.